test/features/support/voucher_activation.js

```diff
@@ -0,0 +1,94 @@
+const pactum = require('pactum');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const { localhost } = require('./helpers/helpers');
+
+let specVoucherActivation;
+
+const baseUrl = `${localhost}vouchers/voucher_activation`;
+const validBodyValue = {
+    "voucher_serial_number": 223,
+    "Gov_Stack_BB": "example Gov_Stack_BB"
+}
```

Comment:
For feature reference: `validBodyValue` is only used once, so no need to put it here as a constant. You can just write in line 36 as:
```
.withBody({
    "voucher_serial_number": 223,
    "Gov_Stack_BB": "example Gov_Stack_BB"
})
```
The same with `invalidSerialNumberBodyValue` and `invalidGovStackBBBodyValue`
:)